unison-cli/src/Unison/CommandLine/Welcome.hs

```diff
@@ -54,7 +54,7 @@ pullBase ns = let
     seg = NameSegment "base"
     rootPath = Path.Path { Path.toSeq = singleton seg }
     abs = Path.Absolute {Path.unabsolute = rootPath}
-    pullRemote = PullRemoteBranchI (Just ns) (Path.Path' {Path.unPath' = Left abs}) SyncMode.Complete Verbosity.Silent
+    pullRemote = PullRemoteBranchI (Just ns) (Path.Path' {Path.unPath' = Left abs}) SyncMode.Complete PullWithoutHistory Verbosity.Silent
```

Comment:
Pull base without history as part of on-boarding, it's faster and cuts down on codebase size.

If folks want to develop on base they need to pull trunk anyways.

Reply:
Let's chat about this, I don't think it's the right approach for slimming base.

Maybe there's a more general preprocessing trick, like `pure . Branch.getAt path` rather than `pure . Branch.discardHistory`?  Or if you wanna get the core of this PR merged and improve the base stuff in a subsequent PR, that's okay too.

Reply:
I reverted it here, I'm still of the opinion we should turn this on, but we can discuss this elsewhere.